### PR TITLE
[WIP] Move to Python 3 & Cleanup

### LINKED ISSUE #282: Switch to Python 3

### PR DESCRIPTION
**Pull request by calixteman**
Maybe I should add tests too here.

Fixes #282.

### CODE REVIEW

**Comment by sylvestre** [comment]
The test suite fails

File: auto_nag/escalation.py [hunk 1]
```diff
@@ -26,8 +26,11 @@ def is_in(self, x):
 
     @staticmethod
     def from_string(s):
-        # TODO: remove this line when switchin to python3
-        s = s.encode('utf-8')
+        # TODO: remove this line when switching to python3
+        try:
+            s = s.encode('utf-8')
+        except:
```
Comment (sylvestre) [inline auto_nag/escalation.py]:
```suggestion
        except Exception:
```

**Comment by sylvestre** [comment]
@calixteman are you still working on this?

**Comment by calixteman** [comment]
@sylvestre yes I'll fix that in the week

**Comment by calixteman** [comment]
I give up this one and I'll made an other one

**Pull request closed without merging**